src/OSPSuite.Presentation/Mappers/PathToPathElementsMapper.cs

```diff
@@ -32,11 +32,6 @@ public PathToPathElementsMapper(IEntityPathResolver entityPathResolver)
       public PathElements MapFrom(IContainer rootContainer, IReadOnlyList<string> entityPath)
       {
          var containers = getContainers(rootContainer, entityPath);
-
-         // Not the whole Path could be found in container hierarchy
-         if (containers.Count < entityPath.Count - 1)
```

Comment:
The needs to be changed so that a path can be retrieved up to all container that are valid. OTherwise I was always getting an empty path